code/modules/admin/verbs/striketeam.dm

```diff
@@ -1,31 +1,46 @@
-//STRIKE TEAMS
+//Deathsquad
 
-#define COMMANDOS_POSSIBLE 6 //if more Commandos are needed in the future
-GLOBAL_VAR_INIT(sent_strike_team, 0)
+#define MAX_COMMANDOS 14
+GLOBAL_VAR_INIT(strike_team_sent, FALSE)
 
 /client/proc/strike_team()
-	if(!SSticker)
-		to_chat(usr, "<span class='userdanger'>The game hasn't started yet!</span>")
+	var/proccaller = usr.client
```

Comment:
```suggestion
	var/client/proccaller = usr.client
```

However, i'm not convinced this is needed at all. You can pass in raw `usr` to `key_name_admin(usr)` just fine with exact same effect as passing in the client; i'm not sure what this var is giving us here.